pages/features.js

```diff
@@ -1,6 +1,10 @@
+/*
+ * Features page
+ */
```

Comment:
I also added a doc block before imports to each file I worked with. I think this is useful, but what do you think?

Reply:
not useful in this way. See my comment above.

Reply:
Agree. This placeholder would be changed. A good example is what these guys use: https://docs.reactioncommerce.com/docs/jsdoc-style-guide#document-a-react-component (but not as complicated, probably only `@file` or `@module` and optional `@link`. I'll make some suggestions...

Reply:
Removed for now.